content/docs/sidebar.json

```
@@ -126,6 +126,167 @@
       }
     ]
   },
+  {
+    "slug": "extensions",
+    "label": "Extensions",
```

Should probably be a section of the User Guide? Also no need for this label (slug is auto-capitalized)
```suggestion
    "slug": "extensions",
```

---

Actually the pages generated look like references ([e.g.](https://mlem-ai-feature-0-3-0-e-qpg1qt.herokuapp.com/doc/extensions/model/sklearn)), it could be called **Extension Reference** (listed under **Command Reference**).

---

> In docs, these items could be subpages of User Guide, e.g.: User Guide > Deployment > Heroku or User Guide > ML frameworks > Onnx.

@aguschin this seems to make sense but the type of content is definitely very much a technical reference so I think it's easier to make a separate section for these extension references.

_BTW I also think current parts of the User Guide like [MLEM ABCs](https://mlem.ai/doc/user-guide/mlem-abcs) are basically references and should be moved/split to their own sections as well._

We can and should still have guides about these topics but without all the specs and formalities, and link to the refs as needed instead. Divide and conquer ⚔️

> merge some pages into what you generate. E.g. ["Working with data"](https://mlem-ai-feature-0-3-0-e-qpg1qt.herokuapp.com/doc/user-guide/data) could be moved here ["Data"](https://mlem-ai-feature-0-3-0-e-qpg1qt.herokuapp.com/doc/extensions/data)

Again this sounds reasonable and if that's the only way to manage things for now (mixing types of documents in different sections) OK but ideally a clear separation between content types goes better with our [docs framework](https://www.notion.so/iterative/Framework-7e86e484e50e42529df3dbfe32b4e987).

---

> re-write some User Guide pages to remove internal details from them...
looking at you, mlem abcs and extending...
moved into new "Objects Reference" page

I do align with these ideas in general @mike0sv. Like I said, separate formalities into reference sections when possible 🙂

---

Ok @jorgeorpinel, looks like we all agree on the summary @mike0sv posted. Some comments of mine above are outdated (written before the summary), so let me just resolve them.